libs/wingsdk/src/shared-aws/service.inflight.ts

```diff
@@ -0,0 +1,49 @@
+import { IServiceClient } from "../cloud";
+
+import { DescribeServicesCommand, ECSClient, UpdateServiceCommand } from "@aws-sdk/client-ecs";
+
+export class ServiceClient implements IServiceClient {
+  constructor(
+    public readonly clusterName: string,
+    public readonly serviceName: string,
+    public readonly client = new ECSClient({})
+  ) {}
+  
+  public async start(): Promise<void> {
+    const command = new UpdateServiceCommand({
+      cluster: this.clusterName,
+      service: this.serviceName,
+      desiredCount: 1,
```

Comment:
I think we need to extend `cloud.Service` to include some scaling parameters.

Reply:
@eladb I went back and forth on adding it, do you think its a functional concern to the developer?